plugins/statistics.koplugin/main.lua

```diff
@@ -1054,6 +1092,80 @@ The max value ensures a page you stay on for a long time (because you fell aslee
                 sub_item_table = self:genResetBookSubItemTable(),
                 separator = true,
             },
+            {
+                text = _("Synchronize now"),
+                callback = function()
+                    SyncService.sync(self.settings.sync_server, db_location, self.onSync )
+                end,
+                enabled_func = function()
+                    return self.settings.sync_server ~= nil and self.settings.is_enabled and require("ui/network/manager"):isWifiOn()
+                end,
+                keep_menu_open = true,
+            },
+            {
+                text_func = function() return _("Cloud sync") end,
```

Comment:
Not fond of seeing that Reading statistics menu grow from 7 items to 9, with these 2 new items I won't care about :)
I guess that for those who will care, `Synchronize now` is worth being accessible quickly - but may be the `Cloud sync` could be moved into the statistics' `Settings>` (I assume it's for setting up where to "Synchronize now", so, visited only once when setting this up?)

Reply:
Done.